src/components/common/eaEmptyState/EAEmptyState.tsx

```diff
@@ -23,18 +24,11 @@ function EAEmptyState({ title, msg, stateType, knowMoreLink, headerText = undefi
                 <div className="fs-20 fw-6 mb-8">{title}</div>
                 <div className="fs-14 m-auto w-600">{msg}</div>
                 <div className='pt-20'>
-                    <a
```

Comment:
Is this intentional?

Reply:
Yes. There will be only one button - `View module` which will redirect to discover module.